.github/workflows/integration_tests.yml

```diff
@@ -73,7 +75,8 @@ jobs:
       PLATFORM: ${{ matrix.PLATFORM }}
       PLATFORM_VERSION: ${{ matrix.PLATFORM_VERSION }}
       KUBECONFIG: ${{ matrix.KUBECONFIG }}
-      KEPTN_NAMESPACE: "keptn-test"
+      KEPTN_NAMESPACE: "keptn-test-${{ github.run_number }}-${{ github.run_attempt }}"
+      KEPTN_NAMESPACE_PREFIX: "keptn-test-${{ github.run_number }}-${{ github.run_attempt }}"
```

Comment:
why the same content for both variables?

Reply:
@mowies same question as @thisthat , otherwise LGTM

Reply:
I had a `-` at the end of the prefix before but then decided to add the `-` where I really need it, so I will remove the second variable and just prefix everything with the namespace instead